package.json

```diff
@@ -52,19 +53,20 @@
   ],
   "license": "Apache-2.0",
   "peerDependencies": {
-    "@hapi/hapi": ">=20.2.1"
+    "@hapi/hapi": "^20.2.1"
   },
   "dependencies": {
     "@hapi/crumb": "^8.0.0",
-    "@hapi/hapi": "^20.2.1",
     "@hapi/hoek": "^9.0.4",
-    "@hapi/inert": "^6.0.3",
```

Comment:
While `@hapi/inert` probably doesn't need to be here, removing it could break someone. Should add it back for now and remove on next breaking version. Remove it from devDependencies too

Reply:
`hapi/inert` is only used in tests; it's fine leaving it in `devDeps`

Reply:
is okay to remove the package the next breaking change? I already push again to dependencies!